app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -442,4 +442,13 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
+
+  <!-- OptionAudioLanguage -->
+  <dimen name="option_audio_language_padding_start">16dp</dimen>
+  <dimen name="option_audio_language_padding_end">16dp</dimen>
+
```

Comment:
@shankarpriyank also this part is maybe copied by you by mistake as you are not using this OptionAudiolanguage dimens anywhere in your files

Reply:
Sorry, I guess this happened while resolving the merge conflicts, I will make fix it

Reply:
Actually this looks a bit weird , my pr which was merged #4220 , I added the dimens for the OptionAudioLanguage  in both the dimen files for the tablet , but right now in the develop branch  I can't see them ,  they were deleted in the last commit through this pr #4226 , I skimmed through it once I feel maybe these values may have been deleted by mistake in that commit

Reply:
@shankarpriyank I actually don't think that there would have been such a big mistake in any other PR else it would not have been merged. I think you should just update your code to develop and keep only the changes required for this PR. PTAL @rt4914

Reply:
okay, I am updating it right now!